webpack/components/extensions/Hosts/BulkActions/BulkRepositorySetsWizard/BulkRepositorySetsWizard.js

```diff
@@ -0,0 +1,167 @@
+import React, { useState, createContext, useContext } from 'react';
+import {
+  Text,
+  TextVariants,
+  TextContent,
+  Wizard,
+  WizardHeader,
+  WizardStep,
+} from '@patternfly/react-core';
+
+import { translate as __ } from 'foremanReact/common/I18n';
+import { useForemanModal } from 'foremanReact/components/ForemanModal/ForemanModalHooks';
+import { useBulkSelect } from 'foremanReact/components/PF4/TableIndexPage/Table/TableHooks';
+import { ForemanActionsBarContext } from 'foremanReact/components/HostDetails/ActionsBar';
+import { useTableIndexAPIResponse, useSetParamsAndApiAndSearch } from 'foremanReact/components/PF4/TableIndexPage/Table/TableIndexHooks';
+
+import { BulkRepositorySetsTable } from './01_BulkRepositorySetsTable';
+import { BulkRepositorySetsReview } from './03_Review';
+import HostReview from '../HostReview';
+import katelloApi from '../../../../../services/api';
+import { useHostsBulkSelect } from '../BulkPackagesWizard/BulkPackagesWizard';
+import { BulkRepositorySetsReviewFooter } from './03_ReviewFooter';
+
+const DEFAULT_PER_PAGE = 5;
+export const BulkRepositorySetsWizardContext = createContext({});
+export const REPO_SETS_URL = katelloApi.getApiUrl(`/repository_sets?per_page=${DEFAULT_PER_PAGE}&include_permissions=true&enabled=true&with_custom=true&organization_id=1`);
```

Comment:
AI found this here https://github.com/parthaa/katello/pull/1/files#r2082470158

`organization_id` is hard coded is that intentional ?

Reply:
It was a leftover, not sure how that didn't break things! updated